Sources/Formworks/FWComponents/FWTextComponent/FWTextModel.swift

```diff
@@ -49,3 +49,5 @@ extension FWTextModel {
         self.isMultiline = dto.isMultiline ?? false
     }
 }
+
+extension FWTextModel: Equatable {}
```

Comment:
Check if this is still needed for the code to work. It is current implemented because the tests require it.